docs/user/installation.rst

```diff
@@ -76,10 +57,48 @@ Alternatively, you can add garage in the pip section of your `environment.yml`
     pip install garage
 
 
+Install System Dependencies (Optional)
```

Comment:
I feel like this section talks about both system deps and installing mujoco stuff, but the heading obviously doesn't convey that. And the content mostly says that you need system deps only for mujoco etc stuff. So maybe we can call this `Optional dependencies`?